tests/Neo.UnitTests/Network/P2P/Payloads/UT_HighPriorityAttribute.cs

```diff
@@ -42,7 +42,7 @@ public void DeserializeAndSerialize()
 
             // Wrong type
 
-            buffer[0] = 0xff;
+            buffer[0] = 0xfe;
```

Comment:
Don´t you want to pick another byte for `ReservedAttribute`?
However, `ff` sound good.
But it could be 0x21 as well.